code/game/objects/items/toys.dm

```diff
@@ -1522,3 +1523,220 @@ var/list/living_balloons = list()
 	B.name = name
 	B.air_contents = air_contents
 	qdel(src)
+
+/*
+ * Action Figures
+ */
+
+/obj/item/toy/figure
+	name = "Non-Specific Action Figure action figure"
```

Comment:
Remove the capitalization here, or give it `\improper`.

Do it for ALL the figures.